<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtClientClass</class>
 <widget class="QMainWindow" name="QtClientClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="FileText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>H:/Lession/code/newcpp/CPPSocket/</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>bytes</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectTxt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>not start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FileSizeTxt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="EndBtn">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>170</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>End Receive</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BuildBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Build TCP connection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Received File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Received File Size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ServerIPTxt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PortEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8099</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtClient.qrc"/>
 </resources>
 <connections/>
</ui>
